backend/src/routes/user-route.js

```diff
@@ -28,6 +28,13 @@ router.post("/login", (req, res) => {
         .catch((err) => standardRouteErrorCallback(res, req, err));
 })
 
+router.post("/signup", (req, res) => {
+    userService.signup(req.body)
+        .then(() => res.status(201).send({}))
+        .catch((err) => standardRouteErrorCallback(res, req, err));
+})
+
+
```

Comment:
There is no such "signup" function for the user service haha. No worries, this part I will handle.